test/manual/definitions/apm/go/go-linux2.json

```diff
@@ -0,0 +1,27 @@
+{
+  "global_tags": {
+    "owning_team": "virtuoso",
+    "Environment": "development",
+    "Department": "product",
+    "Product": "virtuoso"
+  },
+  "resources": [
+    {
+      "id": "go-l2",
+      "provider": "aws",
+      "type": "ec2",
+      "size": "t3.medium",
```

Comment:
@pchinthapenta, I'm curious about the necessity of a medium server for this app. In my opinion, using a small server would be more than sufficient for its needs.

Reply:
changed it to small, also changed the source repo path, it can be merged now